tcms/rpc/tests/test_testexecution.py

```diff
@@ -355,25 +355,64 @@ def test_update_with_single_caserun(self):
         self.assertEqual(execution['sortkey'], 90)
 
     def test_update_with_assignee_id(self):
-        self.assertNotEqual(self.case_run_1.assignee, self.user.pk)
+        self.assertNotEqual(self.case_run_1.assignee, self.user)
         execution = self.rpc_client.TestExecution.update(self.case_run_1.pk, {
             "assignee": self.user.pk
         })
+        self.case_run_1.refresh_from_db()
```

Comment:
You will have to rebase to resolve the conflicts. While at it add another commit which renames `case_run` to `execution` in this entire file.